### ISSUE TITLE
iOS app crashes while attempting to edit logins and Secure notes

### ISSUE THREAD

**Issue by Kabyss** [bitwarden/android#3146]
### Steps To Reproduce

1. Go to 'Logins or Secure notes'
2. Click on 'Edit'

### Expected Result

App should not crash.

### Actual Result

App crashes.

### Additional Context

This happens when logins with larger text entries in the notes section is picked for editing or Secure notes with longer entries are picked for editing. 

### Operating System

iOS

### Operating System Version

17.4.1

### Device

iPhone 13

### Build Version

2024.3.3(7191)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by Neonwarden** [comment]
Hi there,

I attempted to reproduce your issue and was unable to do so. Could you please try uninstalling Bitwarden, restarting your device, and reinstalling the Bitwarden application? 

We use GitHub issues as a place to track bugs and other development-related issues. If your issue persists, please write us back using our “Contact Support” form located on our Help Center (https://bitwarden.com/help/).

You can include a link to this issue in the message content.

Alternatively, you can also search for an answer in our help documentation or get help from other Bitwarden users on our community forums (https://community.bitwarden.com/c/support/). 

The issue here will be closed.

Thanks!

**Comment by Kabyss** [comment]
Noticed something else - issue cannot be reproduced by freshly creating an entry with a random long text. You either need to clone an existing entry that crashes or create a new entry by copy pasting the content of notes section of it.

**Comment by SergeantConfused** [comment]
Hi @Kabyss,

Thank you for following up on this. Going by your last comment, it sounds like there could be something specific in that text that is causing Bitwarden to close unexpectedly; To be clear, I cannot reproduce this with a large Secure Note that was created in the Web App and then accessed in Edit mode on the iOS client.

Could you please create a support ticket so we'd be able to collect more information from you directly? I'm hoping that you see a pattern in the Secure Notes that are causing this; Please remember to include a link to this GitHub report in your message.


Thank you in advance,

**Comment by Kabyss** [comment]
From the Reddit thread, it seems like the character “-“ is the culprit. Can confirm that my issue is also same. 

https://www.reddit.com/r/Bitwarden/s/WFDetk3vRD

**Comment by cbbit** [comment]
Hi @Kabyss 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by Fealthas** [comment]
Same issue. Not sure that - is the problem as I have not that character and yet still crash. 

Pls fix.

**Comment by Kabyss** [comment]
@Fealthas It is possible that “-“ isn’t the only character that triggers it. I would suggest editing the entry and doing a bit of trial and error to see if you can narrow it down to a particular one.

**Comment by Fealthas** [comment]
I also have & @ And .
Otherwise just normal ascii text. 
Can't edit because it crashes hahaha.

**Comment by Kabyss** [comment]
For me, editing from the browser extension or desktop app didn’t result in a crash. I tried with & @ and . but no crash.

**Comment by gpejsa** [comment]
Okay, I finally logged into github as I have had this iOS app crash for some time (months).  I noticed one user above eliminated the "-" from their file.  While I have a bias to using "-" or "--" for certain things in the vault, I went ahead and eliminated them to see if it worked....and it did.  I can now edit and save.  Hard to believe the dash was the culprit, but hey, eliminating it allows me to edit on the app again.
